src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -168,7 +168,22 @@ void Terminal::UpdateAppearance(const ICoreAppearance& appearance)
 {
     _renderSettings.SetRenderMode(RenderSettings::Mode::IntenseIsBold, appearance.IntenseIsBold());
     _renderSettings.SetRenderMode(RenderSettings::Mode::IntenseIsBright, appearance.IntenseIsBright());
-    _renderSettings.SetRenderMode(RenderSettings::Mode::DistinguishableColors, appearance.AdjustIndistinguishableColors());
+
+    switch (appearance.AdjustIndistinguishableColors())
+    {
+    case AdjustTextMode::Always:
+        _renderSettings.SetRenderMode(RenderSettings::Mode::IndexedDistinguishableColors, false);
+        _renderSettings.SetRenderMode(RenderSettings::Mode::AlwaysDistinguishableColors, true);

```

Comment:
```suggestion
        _renderSettings.SetRenderMode(RenderSettings::Mode::IndexedDistinguishableColors, true);
        _renderSettings.SetRenderMode(RenderSettings::Mode::AlwaysDistinguishableColors, true);
```
Should we just set both to true just to be safe? That way if someone in the future adds a feature to indexed, but forgets to check for "always", nothing will break?

Reply:
Yea I'm curious about this too

Reply:
The reason I set this to `false` for now is that we don't want to bother with creating the pre-calculated color table at all if the mode is `Always`, since we don't even access the table and instead we calculate the adjusted foreground just before rendering (see line 255 in RenderSettings.cpp)